.php_cs

```diff
@@ -18,6 +18,7 @@ return PhpCsFixer\Config::create()
         'native_function_invocation' => true,
         'final_class' => true,
         'php_unit_mock_short_will_return' => true,
+        'no_superfluous_phpdoc_tags' => false,
```

Comment:
Disable this rule and it can let `@param` doc comment and native param type hint be existed at same time.

The related issue is available [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4627#issuecomment-584548936).

Reply:
As stated in other comment, it' better to keep this rule and to remove useless phpdoc

Reply:
@garak, let me remove this setting and `php-cs-fixer` will identify useless `phpdoc`.

Reply:
I don't get you. The rule is there just to detect useless phpdoc tags, if you remove it you can't detect it anymore

Reply:
Ah, maybe I see it now: we should keep this rule, but with `true` instead of `false`

Reply:
But I also see that there's no need to keep it with `true`, since it's already included in `@Symfony` rule. So you should remove it